api-catalog-ui/frontend/src/components/Dashboard/Dashboard.jsx

```diff
@@ -88,7 +88,7 @@ export default class Dashboard extends Component {
         }
         return (
             <div className="main-content dashboard-content">
-                <FeedbackButton leftPlacement="80vw" />
```

Comment:
I didn't deploy the UI yet, but I'm not sure about this button usage. The feedback form was currently linked to the `Footer.jsx`, specifically it was openable by clicking on the email icon. Why was it necessary to create a new button for it?